sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -10,7 +10,10 @@ import { PageSettings, PagedAsyncIterableIterator } from "@azure/core-paging";
 import {
   operationOptionsToRequestOptionsBase,
   isTokenCredential,
+  exponentialRetryPolicy,
+  systemErrorRetryPolicy,
 } from "@azure/core-http";
+import { throttlingRetryPolicy } from "../src/policies/throttlingRetryPolicy";
```

Comment:
This import path looks off. Maybe:
```suggestion
import { throttlingRetryPolicy } from "./policies/throttlingRetryPolicy";
```